### PR TITLE
[ADD][9.0] new module hr_payroll_report

### PR DESCRIPTION
**Pull request by Wizr**
which add a pivot view of model `hr.payslip.line`

### CODE REVIEW

**Comment by Wizr** [comment]
cc @elicoidal

File: hr_payroll_report/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Payroll Report",
+    "version": "9.0.0.1.0",
```
Comment (elicoidal) [inline hr_payroll_report/__openerp__.py]: 9.0.1.0.0

File: hr_payroll_report/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "category": "Generic Modules/Human Resources",
```
Comment (elicoidal) [inline hr_payroll_report/__openerp__.py]: Add a summary key

File: hr_payroll_report/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "website": "https://odoo-community.org/",
+    "author": "Elico Corp (www.elico-corp.com), "
```
Comment (elicoidal) [inline hr_payroll_report/__openerp__.py]: remove " (www.elico-corp.com)"

File: hr_payroll_report/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Payroll Report",
+    "version": "9.0.0.1.0",
+    "category": "Generic Modules/Human Resources",
+    "website": "https://odoo-community.org/",
```
Comment (elicoidal) [inline hr_payroll_report/__openerp__.py]: https://www.elico-corp.com

File: hr_payroll_report/README.rst [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+This allows to create all kind of analysis for the payroll department
+
+Installation
+============
+
+To install this module, you need to:
+
+* have basic module `hr_payroll` installed.
+
+Bug Tracker
```
Comment (elicoidal) [inline hr_payroll_report/README.rst]: I miss the runbot section

**Review by elicoidal — CHANGES_REQUESTED** [review]
some details

**Review by elicoidal — APPROVED** [review]

**Comment by elicoidal** [comment]
@feketemihai @dufresnedavid your review is welcome

**Comment by elicoidal** [comment]
@feketemihai @dufresnedavid is there any interest in this module?

**Comment by feketemihai** [comment]
@elicoidal @Wizr Thanks for the module, it's o aggod one, but i have some remarks to do, since there is a little code in in can you update to new api, second, can we add a field in salary rules to be showned on this report, i know we already have one for showing in to the payslip, but to have a good report we should be filtering the gross and the net lines, regarding the total amount, ot to add in readme this conclusion, so everybody can add a default filter not to show this lines. Regarding the measures i think that they are too much, i think the first 4 it's enough, since you will probably want to make some comparition between payslips, and if you want to detail on one you can add the measures after...

**Comment by elicoidal** [comment]
@feketemihai thanks for your feedback

> since there is a little code in in can you update to new api

:+1: 

> second, can we add a field in salary rules to be showned on this report, i know we already have one for showing in to the payslip, but to have a good report we should be filtering the gross and the net lines, regarding the total amount, ot to add in readme this conclusion, so everybody can add a default filter not to show this lines.

:-1:  I think it is dependent on every HR manager, country etc. With  current solution, we can be flexible without complexity

> Regarding the measures i think that they are too much, i think the first 4 it's enough, since you will probably want to make some comparition between payslips, and if you want to detail on one you can add the measures after...

I'd rather keep them if necessary (it is useful for some analysis) but not display in the table by default. What do you think?

**Comment by feketemihai** [comment]
@elicoidal @Wizr I'm fine with your comments, and to adapt the last one, not to be shown by default all measures, so you have my :+1:  Can you adapt it so i approve the PR?

**Comment by elicoidal** [comment]
@wizr can you please update the code accordingly?

**Review by feketemihai — APPROVED** [review]

**Comment by elicoidal** [comment]
I can safely merge

**Pull request merged**